framework/web/ErrorAction.php

```diff
@@ -98,6 +98,13 @@ public function init()
      */
     public function run()
     {
+        $response = Yii::$app->getResponse();
+        if ($this->exception instanceof HttpException) {
```

Comment:
to fix the duplication of logic in:
* https://github.com/yiisoft/yii2/pull/13708/files#diff-f4cd9aa0a2255e5362d10b956fe70974R102
* https://github.com/yiisoft/yii2/pull/13708/files#diff-7a2fbae94c890a0d999b38f8a82fe81eR92